src/main/java/com/bogging/system/Blogging/System/Services/SubcategoryService.java

```diff
@@ -0,0 +1,50 @@
+package com.bogging.system.Blogging.System.Services;
+
+
+import com.bogging.system.Blogging.System.Entity.Category;
+import com.bogging.system.Blogging.System.Entity.Subcategory;
+import com.bogging.system.Blogging.System.ExceptionHandling.ResourceNotFoundException;
+import com.bogging.system.Blogging.System.Repository.CategoryRepository;
+import com.bogging.system.Blogging.System.Repository.SubcategoryRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class SubcategoryService {
+
+    @Autowired
+    private SubcategoryRepository subcategoryRepository;
+
+    @Autowired
+    private CategoryRepository categoryRepository;
```

Comment:
Remove category repository and use services communication

Reply:
I have done some steps using feign client dependency but there are need some help to register services on ureka or any other concept will required.